lib/fshash/bucket_memory.go

```diff
@@ -0,0 +1,51 @@
+package fshash
+
+import (
+	"sort"
+	"strings"
+
+	"polydawn.net/repeatr/lib/treewalk"
+)
+
+var _ Bucket = &MemoryBucket{}
+
+type MemoryBucket struct {
```

Comment:
Thoughts on moving MemoryBucket to a separate package?

Would enforce the comment I made above about implementations + imports, etc.